src/servers/src/http.rs

```diff
@@ -127,6 +127,8 @@ pub struct HttpOptions {
     pub disable_dashboard: bool,
 
     pub body_limit: ReadableSize,
+
+    pub strict_mode: bool,
```

Comment:
this is a matter of preference/personal taste but from my perspective having name `is_strict_mode` suits better for booleans. Especially, in places like 
```
    is_zstd: bool,
    body: Body,
    is_strict_mode: bool,
```